README.md

```diff
@@ -6,11 +6,25 @@ UCLA Library Management -- Tenejo
 Development
 -----------
 
+1. Clone the project repository:
+   `git clone https://github.com/UCLALibrary/samvera-mgmt.git; cd samvera-mgmt`
+1. Install dependencies
+   `bundle install`
 1. Setup your database.
    We use PostgreSQL. To support the test and development environments, you'll
    need have Postgres installed and running. In your `psql` console do
    `create role tenejo with createdb login`. Then do
```

Comment:
this is kinda pedantic, I know, but you should close your SQL statement with a semicolon, or it won't run

Reply:
:+1:

Reply:
Thanks @hardyoyo. I updated and force pushed the change.